bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```
@@ -14,6 +14,9 @@ metadata:
     app.kubernetes.io/component: pgpool
   {{- if .Values.commonAnnotations }}
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- if and (not .Values.pgpool.customUsersSecret) .Values.pgpool.customUsers.usernames }}
+    checksum/custom-users-secrets: {{ include (print $.Template.BasePath "/custom-users-secrets.yaml") . | sha256sum }}
```

Isn't it `/pgpool/custom-users-secrets.yaml` the correct path?

```suggestion
    checksum/custom-users-secrets: {{ include (print $.Template.BasePath "/pgpool/custom-users-secrets.yaml") . | sha256sum }}
```